src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -2312,6 +2312,7 @@ internal static bool PreventsSuccessfulDelegateConversion(ErrorCode code)
                 case ErrorCode.ERR_QueryRangeVariableSameAsTypeParam:
                 case ErrorCode.ERR_DeprecatedCollectionInitAddStr:
                 case ErrorCode.ERR_DeprecatedSymbolStr:
+                case ErrorCode.ERR_MissingPredefinedMember:
```

Comment:
> Hmm. I'm not sure about this change. This will impact lambdas besides ones with default parameters or params arrays. I think we convert in the face of this error, and then report the error when encountered.

This is what this change is doing, i.e. doesn't block the conversion from happening. Let's discuss this offline, I would like to get a better understanding of your concern.

Reply:
Alright, rereading the change I misread the false as true, and it's behaving exactly as I expect. It is still a change that is observable to other lambdas, though, so I think we need to give compat council a heads up (even though the likelihood of this being observable is vanishingly small).

Reply:
OK, I can send an email before this is merged. But I think this change results only in different errors, so it doesn't change any successfully-compiling code, or am I wrong? Also note that there's a follow-up https://github.com/dotnet/roslyn/issues/66060 to add more errors here.

Reply:
> But I think this change results only in different errors, so it doesn't change any successfully-compiling code, or am I wrong?

Yes, it will change successfully compiling code into an ambiguity error. It may be very hard to come up with an example that observes the change, but it should be possible.